### DISCUSSION TITLE
syntax highlighting for html and css possible?

### DISCUSSION THREAD

**Discussion by BindingOx** [BryanJenksCommunity/FAQ#145] [Q&A] [answered]
I see that 

```javascript
var test = x
```
shows syntax highlighting in anki but not from something like:
```css
h1 > a {
}
```

Can we change this?

**Comment by tallguyjenks** [comment, marked as answer]
theres anki add ons and [things like this](https://www.alveeakand.com/anki-syntax-highlighting/) to keep things simple i just took screenshots of code in carbon.sh

**Reply by BindingOx** [reply]
Thanks for letting me know about carbon.sh. I'll work this into my workflow because although addons like [Syntax Highlighting for Code - AnkiWeb](https://ankiweb.net/shared/info/1463041493) work, if I change the card in Obsidian and resync, then the styling reverts back to its ugly default.  Much easier to deal with screenshots :D
